README.md

```diff
@@ -61,6 +66,13 @@ chown root: certbot_zimbra.sh
 mv certbot_zimbra.sh /usr/local/bin/
 ```
 Or from the master branch: [certbot_zimbra.sh](/../../raw/master/certbot_zimbra.sh)
+```bash
+wget -O /usr/local/bin/certbot_zimbra.sh https://github.com/YetOpen/certbot-zimbra/raw/master/certbot_zimbra.sh;
+chmod +x /usr/local/bin/certbot_zimbra.sh;
+wget -O /etc/cron.d/zimbracrontab https://github.com/YetOpen/certbot-zimbra/raw/masterzimbracrontab;
+chmod +x /etc/cron.d/zimbracrontab
```

Comment:
Added these snippets from Readme to the repo so they can be easily updated in future without lots of manual editing or stuff like the below.

```
cat >> /etc/cron.d/zimbracrontab <<-EOL
# certbot_zimbra.sh requires bash and a path with /usr/sbin
SHELL=/bin/bash
PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin

# Replace /usr/bin/certbot with the location of your certbot binary, use this to find it: which certbot-auto certbot letsencrypt
12 5 * * * root /usr/bin/certbot renew --pre-hook "/usr/local/bin/certbot_zimbra.sh -p" --deploy-hook "/usr/local/bin/certbot_zimbra.sh -d"
EOL
```